packages/dev/parcel-namer-docs/DocsNamer.js

```diff
@@ -37,11 +37,12 @@ module.exports = new Namer({
       );
     } else if (!bundle.target || !bundle.target.distEntry) {
       // An asset. Should end up hashed in the root.
-      let bundleGroup = bundleGraph.getBundleGroupsContainingBundle(bundle)[0];
-      let bundleGroupBundles = bundleGraph.getBundlesInBundleGroup(bundleGroup);
-      let mainBundle =  bundleGroupBundles.find(b => b.getEntryAssets().some(a => a.id === bundleGroup.entryAssetId));
-      let entry = mainBundle.getEntryAssets().find(a => a.id === bundleGroup.entryAssetId).filePath;
-      return path.basename(entry, path.extname(entry)) + '.' + bundle.hashReference + '.' + bundle.type;
+      // let bundleGroup = bundleGraph.getBundleGroupsContainingBundle(bundle)[0];
```

Comment:
deleted commented out code?

Reply:
Ah, oops. Looks like we don't need these changes. Must have been debugging.